language/en/info_acp_ideas.php

```diff
@@ -19,5 +19,18 @@
 }
 
 $lang = array_merge($lang, array(
-	'LOG_IDEA_TITLE_EDITED'		=> '<strong>Idea title edited</strong><br />» Idea ID #%s',
+	// ACP module
```

Comment:
All this module stuff should go in it's own language file that is loaded by the module. Only stuff that should go in an info_acp_ file are global lang vars, which basically means LOG lang vars and ACP MODULE name lang vars (see one of our other exts as example).